### PR TITLE
fix: add 'kubectl' to the image

### PR DESCRIPTION
**Pull request by Roming22**
[RHTAPBUGS-1317](https://issues.redhat.com//browse/RHTAPBUGS-1317)

Also sneaking a couple of cli output improvements.

### CODE REVIEW

**Review by flacatus — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -26,10 +26,11 @@ FROM registry.access.redhat.com/ubi9-minimal:9.4-1227
 
 WORKDIR /rhtap-cli
 
+COPY --from=quay.io/codeready-toolchain/oc-client-base:latest /usr/bin/kubectl /usr/bin/
```
Comment (otaviof) [inline Dockerfile:31]: Please use `microdnf` to install `kubectl` instead.

**Review by otaviof — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (Roming22) [reply Dockerfile:31]: I couldn't find the package in `microdnf`.

Using
```
RUN curl --proto "=https" -L "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/openshift-client-linux.tar.gz" -o /tmp/openshift-client-linux.tar.gz && \
    tar --no-same-owner -xzf /tmp/openshift-client-linux.tar.gz && \
    mv oc kubectl /usr/local/bin && \
    oc version --client && \
    kubectl version --client
```
fails the build of non amd64 images.

Comment (Roming22) [reply Dockerfile:31]: I'll leave it this way for the moment so we can release. We can try to do it better in a future release.

**Review by jduimovich — APPROVED** [review]

**Comment by flacatus** [comment]
@Roming22 feeel free to merge. The test failure is not related with ur changes

**Pull request merged**